sdk/storage/azblob/zt_examples_test.go

```diff
@@ -201,182 +183,175 @@ func ExampleStorageError() {
 
 // This example demonstrates splitting a URL into its parts so you can examine and modify the URL in an Azure Storage fluent way.
 func ExampleBlobURLParts() {
-	// Let's begin with a snapshot SAS token.
+	// Here is an example of a blob snapshot.
 	u := "https://myaccount.blob.core.windows.net/mycontainter/ReadMe.txt?" +
 		"snapshot=2011-03-09T01:42:34Z&" +
 		"sv=2015-02-21&sr=b&st=2111-01-09T01:42:34.936Z&se=2222-03-09T01:42:34.936Z&sp=rw&sip=168.1.5.60-168.1.5.70&" +
 		"spr=https,http&si=myIdentifier&ss=bf&srt=s&sig=92836758923659283652983562=="
 
 	// Breaking the URL down into it's parts by conversion to BlobURLParts
-	parts := NewBlobURLParts(u)
-
-	// Now, we can access the parts (this example prints them.)
-	fmt.Println(parts.Host, parts.ContainerName, parts.BlobName, parts.Snapshot)
-	sas := parts.SAS
-	fmt.Println(sas.Version(), sas.Resource(), sas.StartTime(), sas.ExpiryTime(), sas.Permissions(),
-		sas.IPRange(), sas.Protocol(), sas.Identifier(), sas.Services(), sas.Signature())
-
-	// You can also alter some of the fields and construct a new URL:
-	// Note that: SAS tokens may be limited to a specific container or blob.
-	// You should be careful about modifying SAS tokens, as you might take them outside of their original scope accidentally.
-	parts.SAS = SASQueryParameters{}
+	parts := azblob.NewBlobURLParts(u)
+
+	// The BlobURLParts allows access to individual portions of a Blob URL
+	fmt.Printf("Host: %s\nContainerName: %s\nBlobName: %s\nSnapshot: %s\n", parts.Host, parts.ContainerName, parts.BlobName, parts.Snapshot)
+	fmt.Printf("Version: %s\nResource: %s\nStartTime: %s\nExpiryTime: %s\nPermissions: %s\n", parts.SAS.Version(), parts.SAS.Resource(), parts.SAS.StartTime(), parts.SAS.ExpiryTime(), parts.SAS.Permissions())
+
+	// You can alter fields toconstruct a new URL:
+	// Note: SAS tokens may be limited to a specific container or blob, be careful modifying SAS tokens, you might take them outside of their original scope accidentally.
+	parts.SAS = azblob.SASQueryParameters{}
 	parts.Snapshot = ""
 	parts.ContainerName = "othercontainer"
 
 	// construct a new URL from the parts
 	fmt.Print(parts.URL())
 }
 
-// This example demonstrates how to use the SAS token convenience generators.
-// Though this example focuses on account SAS, these generators exist across all clients (Service, Container, Blob, and specialized Blob clients)
+// This example demonstrates how to use the SAS token generators, this example focuses on account SAS,
+// these generators exist across all clients (Service, Container, Blob, and specialized Blob clients)
 func ExampleServiceClient_GetSASToken() {
-	// Initialize a service client
```

Comment:
Why remove this comment?

Reply:
The service client wasn't being initialized in the line below and I don't think that we need to clarify that portion.